guide/messaging-dispatcher-service/messaging-dispatcher.bal

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/jms;
+import ballerina/log;
+import ballerina/mb;
+import ballerina/io;
+
+endpoint http:Client  alarm1 {
+    url: "http://localhost:8080/alarm1"
+};
+endpoint http:Client  alarm2 {
+    url: "http://localhost:8080/alarm2"
+};
+endpoint http:Client  alarm3 {
+    url: "http://localhost:8080/alarm3"
+};
+endpoint jms:SimpleTopicSubscriber subscriberSetAlarmOff {
+    initialContextFactory: "bmbInitialContextFactory",
+    providerUrl: "amqp://admin:admin@carbon/carbon?"
+        + "brokerlist='tcp://localhost:5672'",
```

Comment:
Can merge with the line above.

Reply:
Merge with the line above it and format.